docs/maps.rst

```diff
@@ -2,5 +2,139 @@ Maps (``datascience.maps``)
 ===========================
 
 .. automodule:: datascience.maps
-    :members:
-    :undoc-members:
+
+**Summary of Methods for FoliumWrapper**
+
+The folium wrapper class creates a map element that can be drawn. It takes an abstract base class as an argument.
+    
+Display
+
+.. autosummary::
+    :toctree: _autosummary
+    
+    _FoliumWrapper.draw
```

Comment:
What is the rationale for including docs on _FoliumWrapper?  Normally classes that start with an underscore are not intended to be a primary part of the visible API?